airbyte-integrations/connectors/source-waiteraid/source_waiteraid/waiteraid.yaml

```diff
@@ -0,0 +1,48 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  requester:
+    url_base: "https://app.waiteraid.com"
+    http_method: "POST"
+    request_options_provider:
+      request_parameters:
+        date: "{{ config['start_date'] }}"
+        auth_hash: "{{ config['auth_hash'] }}"
+        restid: "{{ config['restid'] }}"
+  stream_slicer:
+    type: "DatetimeStreamSlicer"
+    start_datetime:
+      datetime: "{{ config['start_date'] }}"
+      datetime_format: "%Y-%m-%d"
+    end_datetime:
+      datetime: "{{ now_utc() }}"
+      datetime_format: "%Y-%m-%d %H:%M:%S.%f"
+    step: "1d"
+    datetime_format: "%Y-%m-%d"
+    cursor_field: "{{ options['stream_cursor_field'] }}"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
+    requester:
+      $ref: "*ref(definitions.requester)"
+  base_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+  booking_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "booking"
+      path: "/wa-api/searchBooking"
+      stream_cursor_field: "date"
+
+streams:
+  - "*ref(definitions.booking_stream)"
```

Comment:
Do you plan to add others streams like restaurants, meals, meal status...?

Reply:
Not really, since most of the info is either already included in the booking stream or doesn't provide much additional insight. 
For example restaurants stream gives you a list of restaurants that you have access to.  The connector is written to be used for one restaurant at a time (similar to the Facebook source) so this stream isn't needed.
Meals returns hours when meals are served, like breakfast from 8 to 10, lunch from 11 to 1 etc which doesn't provide business insight. 
Meal status provides info on available time slots for a booking  for one single week only. This info can be derived from information in the booking stream.

There are also a few streams for adding or modifying information.